<!-- -->

<launch>

	<node pkg="image_saver" type="image_saver_node" name="image_saver_node" output="screen" >
		<param name ="image_topic" value="image_raw"/>
		<param name ="image_path" value="/home"/>
		<param name ="use_trigger" value="true"/>
	</node>

</launch>

